.github/workflows/esy-test.yml

```diff
@@ -0,0 +1,42 @@
+name: Pollinate tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+  schedule:
+    # Prime the caches every Monday
+    - cron: 0 1 * * MON
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
```

Comment:
@ulrikstrid I think it exists a better way to only use one `os`, right?

Reply:
Yeah, just remove the matrix all together, the matrix is only useful if you want to test multiple combinations